course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -198,7 +198,7 @@
             var searchTermList = $scope.getSearchTermList(searchTerms);
             // open a modal confirmation box and as the user to verify they want to add
             // the number of users they entered.
-            var modalInstance = $uibModal.open({
+            $scope.modalInstance = $uibModal.open({
```

Comment:
If we're going to put this on the scope, can we call it something more specific, like `confirmAddModal`?